src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Stylesheets with an SCSS extension should not be linked directly in HTML. You should compile SCSS to CSS and link the CSS file instead.